storybook/preview-head.html

```diff
@@ -1,11 +1,14 @@
 <meta name="viewport" content="width=device-width, initial-scale=1" />
 
 <link rel="preconnect" href="https://fonts.gstatic.com" />
-<link href="https://fonts.googleapis.com/css2?family=Source+Code+Pro:ital,wght@0,400;0,600;1,400&display=swap" rel="stylesheet" />
+<link
+  href="https://fonts.googleapis.com/css2?family=Source+Code+Pro:ital,wght@0,400;0,600;1,400&display=swap"
+  rel="stylesheet"
+/>
 
 <style type="text/css">
   html {
-    font-family: 'Akzidenz', Helvetica, Arial, sans-serif;
+    font-family: 'Euclid', 'Akzidenz', Helvetica, Arial, sans-serif;
```

Comment:
Are we able to do this selectively? Not all components are going to be able to use this yet.